[json-schema-validator] wrong cmake install targets

### Is your feature request related to a problem? Please describe.

When linking against `json-schema-validator` on windows using the current port verion `2.3.0` the library cannot be resolved as the project is not configured properly.

The fix for this is already on `main` but hasn't received a new patch version yet.
https://github.com/pboettch/json-schema-validator/commit/0034c113477f83c28d4380de1ee189c25b1168e6

### Proposed solution

I propose to evaluate whether it makes sense to provide a manual patch for this version of the library or waiting the patch realease and simply bump version.

### Describe alternatives you've considered

_No response_

### Additional context

```cmake
find_package(nlohmann_json_schema_validator REQUIRED)
target_link_libraries(main PRIVATE nlohmann_json_schema_validator)
```
Gives the following output when trying to link:
```sh
ld.exe: cannot find -lnlohmann_json_schema_validator: No such file or directory
```

Further investigating the issue I stumbled across `linkLibs.rsp` generated by my mingw toolchain with the following content (shortend):

```bash
... "vcpkg_installed/x64-mingw-static/lib/libnlohmann_json_schema_validator.a" ... "vcpkg_installed/x64-mingw-static/lib/libfmt.a" -lnlohmann_json_schema_validator ... -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 
```

Notice the `-lnlohmann_json_schema_validator`...

Finding `libnlohmann_json_schema_validator.a` works just fine...

This is where my linking error is coming from:

```
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lnlohmann_json_schema_validator: No such file or directory
collect2.exe: error: ld returned 1 exit status
mingw32-make[3]: *** [CMakeFiles\ocxreader.dir\build.make:155: app.exe] Error 1
mingw32-make[2]: *** [CMakeFiles\Makefile2:134: CMakeFiles/app.dir/all] Error 2
mingw32-make[1]: *** [CMakeFiles\Makefile2:141: CMakeFiles/app.dir/rule] Error 2
mingw32-make: *** [Makefile:198: app] Error 2
```

--- a/ports/json-schema-validator/portfile.cmake
+++ b/ports/json-schema-validator/portfile.cmake
@@ -21,6 +21,6 @@ vcpkg_copy_pdbs()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
-file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
 file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 

--- a/ports/json-schema-validator/usage
+++ b/ports/json-schema-validator/usage
@@ -1,4 +1,4 @@
-The package json-schema-validator provides CMake integration:
+json-schema-validator provides CMake integration:
 
     find_package(nlohmann_json_schema_validator REQUIRED)
-    target_link_libraries(main PRIVATE nlohmann_json_schema_validator)
+    target_link_libraries(main PRIVATE nlohmann_json_schema_validator::validator)

--- a/ports/json-schema-validator/vcpkg.json
+++ b/ports/json-schema-validator/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "json-schema-validator",
   "version": "2.3.0",
+  "port-version": 1,
   "description": "C++ library for validating JSON documents based on a JSON Schema. This validator is based on the nlohmann-json library.",
   "homepage": "https://github.com/pboettch/json-schema-validator",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3706,7 +3706,7 @@
     },
     "json-schema-validator": {
       "baseline": "2.3.0",
-      "port-version": 0
+      "port-version": 1
     },
     "json-spirit": {
       "baseline": "4.1.0",

--- a/versions/j-/json-schema-validator.json
+++ b/versions/j-/json-schema-validator.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "31c71975b94068010a3985ac84deed50047c6820",
+      "version": "2.3.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "4466bee34b5544a3ecfaf5f480464b1f9d45e9b1",
       "version": "2.3.0",
       "port-version": 0

